### PR TITLE
Update docs how to create a Cosmos DB for NoSQL account

### PR DESCRIPTION
**Pull request by kota2and3kan**
Note: This PR depends on PR #813.

This PR updates the steps of creating/configuring the `Cosmos DB for NoSQL` account for ScalarDB in the getting started guide.
Before, the getting started guide explains concrete steps based on the Azure portal page, but there is a possibility that the UI of the portal page will be changed in the future.
So, I updated it to describe the overview of steps and add links to the official document.
This can reduce the maintenance cost of documents when the Azure portal UI is changed.

Please take a look!

### CODE REVIEW

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you!

File: docs/getting-started-with-scalardb-on-cosmosdb.md [hunk 1]
```diff
@@ -13,23 +13,10 @@ ScalarDB is written in Java. So the following software is required to run it.
...
-    * Select **Review + create**. You can skip the **Network** and **Tags** sections.
-    * Review the account settings, and then select **Create**.
-    *  Wait some time for **Azure Cosmos DB** account creation.
- 4. Select **Go to resource** to go to the Azure Cosmos DB account page.
- 5. Select **Default consistency** from the left navigation on your Azure Cosmos DB account page.
-    * Change `Consistency Level` from `SESSION` to `STRONG`.
-    * Select **Save**
-        
-From here, we assume Oracle JDK 8 is properly installed in your local environment and the Azure Cosmos DB account is properly configured in Azure.
+1. Create Cosmos DB for NoSQL account according to the official document [Create an Azure Cosmos DB account](https://learn.microsoft.com/en-us/azure/cosmos-db/nosql/quickstart-portal#create-account)
```
Comment (komamitsu) [inline docs/getting-started-with-scalardb-on-cosmosdb.md:16]: > Create Cosmos DB for NoSQL account

`Create a Cosmos DB for NoSQL account` would be slightly better?

**Review by komamitsu — APPROVED** [review]
LGTM, thank you!

File: docs/getting-started-with-scalardb-on-cosmosdb.md (hunk 1 above)
Comment (kota2and3kan) [reply docs/getting-started-with-scalardb-on-cosmosdb.md]: I agree. I updated it.
Thank you!

**Pull request merged**